src/components/Question.spec.js

```diff
@@ -0,0 +1,64 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import Question from './Question'
+
+describe('Question', () => {
+  it('should render a question, options and a button', () => {
+    render(
+      <Question
```

Comment:
Test fails with prop types warnings, you forgot to add `onChange` and `onUpdateSteps`.. Maybe think about adding CI pipeline to automatically run your tests on push?

Reply:
That's a really good idea! Thanks 👍